src/site/content/en/fast/using-bundlesize-with-travis-ci/index.md

```diff
@@ -38,8 +38,9 @@ kitty](https://glitch.com/edit/#!/scarce-pixie).
 ## Set the performance budget
 
 [This Glitch](https://glitch.com/edit/#!/scarce-pixie) already contains
-bundlesize. To start, click the **Remix to Edit** button to make the project
-editable.
+bundlesize.
+
+{% Instruction 'remix' %}
```

Comment:
Should this be set to list-style none? A single bullet looks strange on its own.

Reply:
Good question. In the past I've used the bullet style to call something out so folks recognize that a step needs to happen (in case they're skimming). I could go either way I guess.

Reply:
The stickler part of my brain is reminding me that things like lists and menus should have at least two items, but I get the salience argument. And it occurs to me that I failed to comment on other instances of singleton bullets, so let's not worry about it :)